manifests/j/JanDeDobbeleer/OhMyPosh/3.162.0/JanDeDobbeleer.OhMyPosh.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate
+
+PackageIdentifier: JanDeDobbeleer.OhMyPosh
+PackageVersion: 3.162.0
+InstallModes:
+- silent
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v3.162.0/install.exe
+  InstallerSha256: 321AC6D25B63F81E7511D89CCE3D5BF83BF0D5FBCCE215B39A07EFE7F82FC2F1
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v3.162.0/install.exe
+  InstallerSha256: 321AC6D25B63F81E7511D89CCE3D5BF83BF0D5FBCCE215B39A07EFE7F82FC2F1

```

Comment:
```suggestion
- Architecture: x86
  InstallerType: inno
  InstallerUrl: https://github.com/JanDeDobbeleer/oh-my-posh/releases/download/v3.162.0/install.exe
  InstallerSha256: 321AC6D25B63F81E7511D89CCE3D5BF83BF0D5FBCCE215B39A07EFE7F82FC2F1
```

Reply:
Question about this though. Does the architecture point to the installer architecture or eventual installed file architecture? The reason I ask is that the Inno installer contains both an x64 and x86 executable.

Reply:
The `InstallerUrl` for both architectures seem to be the same; so keeping only the `x86`  architecture will still allow it to run on arm64, x64 and x86 devices.

Reply:
@ItzLevvie for me that would be `C:\Users\jan\AppData\Local\Programs\oh-my-posh`

Reply:
> if there's any switches you can provide

@ItzLevvie that's basically what the inno installers does for me.